p2p/src/swarm/mod.rs

```diff
@@ -14,46 +14,33 @@
 // limitations under the License.
 //
 // Author(s): A. Altonen
+
+//! Peer manager
+//!
+//! TODO
+//!
+//!
+
+#![allow(rustdoc::private_intra_doc_links)]
 use crate::{
-    error::{FatalError, P2pError, PeerError, ProtocolError},
+    error::{P2pError, PeerError, ProtocolError},
     event,
     net::{self, ConnectivityService, NetworkingService},
 };
-use common::chain::ChainConfig;
+use common::{chain::ChainConfig, primitives::version};
```

Comment:
We should probably rename `primitives::version` to `primitives::semver`, so it is not confused with the newly introduced `primitives::version_tag`

Reply:
Done